evap/results/tests/test_exporters.py

```diff
@@ -108,12 +108,12 @@ def test_view_excel_file_sorted(self):
 
         # Load responses as Excel files and check for correct sorting
         workbook = xlrd.open_workbook(file_contents=content_de.read())
-        self.assertEqual(workbook.sheets()[0].row_values(0)[1], "A - Evaluation1")
-        self.assertEqual(workbook.sheets()[0].row_values(0)[2], "B - Evaluation2")
+        self.assertEqual(workbook.sheets()[0].row_values(0)[1], "A – Evaluation1")
```

Comment:
I already know that I will use a "-" at some point in the future. I'm sorry, in advance :laughing: